### PR TITLE
[FR] Add `News_Styling_Criteria`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page `News_Styling_Criteria`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

*Feel free to merge if the delay is too long for a review by a native.*

### CODE REVIEW

File: wiki/News_Styling_Criteria/fr.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+Le texte alternatif ne doit pas être utilisé dans les bannières.
+
+### HTML et contenu intégré
+
+L'utilisation limitée de HTML dans le but d'intégrer du contenu hors site tel que des vidéos YouTube, des VODs Twitch ou d'autres applets conçus pour mettre en valeur le contenu du site d'osu! ou du site d'osu! est autorisée.
+
+L'utilisation de la balise ``<center>`` (ou tout autre style équivalent tel que ``<p align='center'>``) n'est pas autorisée car le style du système de news centre automatiquement les éléments non textuels.
+
+La largeur de tous les cadres de contenu intégré doit être fixée à ``width: 100%``. Pour être inclus dans les articles en général, le contenu intégré doit pouvoir s'afficher en pleine largeur sans se briser ou avoir l'air affreux.
```
Comment (Satsukel) [inline wiki/News_Styling_Criteria/fr.md:139]:
```suggestion
La largeur de tous les cadres de contenu intégré doit être fixée à ``width: 100%``. Pour être inclus dans les articles en général, le contenu intégré doit pouvoir s'afficher en pleine largeur sans problème ou sans être horrible.
```

**Review by Satsukel — CHANGES_REQUESTED** [review]

**Comment by TicClick** [comment]
while the initiative is good, I highly doubt we'll ever have news in any language other than English

**Comment by DoryanR** [comment]
I am aware of this, but I have translated the article for information purposes. I will add a mention at the top of the article about this point.

File: wiki/News_Styling_Criteria/fr.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
+# Critères de style des news
+
+*Pour les articles du wiki, voir : [Critères de mise en forme des articles](/wiki/Article_styling_criteria)*
+
+Les articles de news existent sur un système légèrement différent du wiki principal d'osu!, mais suivent des principes similaires. Leur rédaction implique une attention particulière à quelques caractéristiques et formats clés, comme indiqué ci-dessous.
```
Comment (DoryanR) [inline wiki/News_Styling_Criteria/fr.md:5]:
```suggestion
# Critères de style des news

*Pour les articles du wiki, voir : [Critères de mise en forme des articles](/wiki/Article_styling_criteria)*

**Attention : les articles de news sont disponible uniquement en Anglais, et une traduction de ces articles dans d'autres langues n'est pas prévu.**

Les articles de news existent sur un système légèrement différent du wiki principal d'osu!, mais suivent des principes similaires. Leur rédaction implique une attention particulière à quelques caractéristiques et formats clés, comme indiqué ci-dessous.
```

File: wiki/News_Styling_Criteria/fr.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+### Noms de fichiers d'articles
+
+Les articles de news sont des fichiers markdown (``.md``) nommés selon le format suivant :
+
+``yyyy-mm-dd-<article title>.md``
+
+Où `yyyy` est l'année à 4 chiffres, `mm` est le mois à 2 chiffres, et `dd` est la date à 2 chiffres.
+
+Les chaînes de titre d'article pour les noms de fichier sont le titre de l'article lui-même avec tous les espaces remplacés par des traits d'union (`-`), et tous les caractères supplémentaires tels que les points (`.`), les points d'exclamation (`!`), les points d'interrogation (` ? `), les virgules (`,`), les apostrophes (`'`), les guillemets (`"`), moins que (`<`), plus que (`>`), les deux-points (`:`), les barres obliques (`/`), les barres obliques (`\`), les barres verticales (`|`) et les astérisques (`*`) sont retirés.
```
Comment (Biokax) [inline wiki/News_Styling_Criteria/fr.md:53]: le début de la phrase manque de sens, j'ai du mal à la comprendre

File: wiki/News_Styling_Criteria/fr.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+Les articles de news n'utilisent pas les titres Markdown traditionnels, à quelque titre que ce soit.
+
+#### En-têtes
+
+Les niveaux d'en-tête 1 (``#``), 2 (``##``) et 3 (``###``) peuvent être utilisés. Ils ont des identifiants uniques et peuvent être liés directement. N'utilisez jamais les en-têtes pour styliser ou formater du texte.
+
+#### Gras
+
+Les caractères gras sont réservés à la mise en évidence d'objets ou de phrases importants, qu'il s'agisse de joueurs, de scores ou de tout autre élément considéré contextuellement comme un point central de l'article en question.
```
Comment (Biokax) [inline wiki/News_Styling_Criteria/fr.md:87]:
```suggestion
Les caractères gras sont réservés à la mise en évidence d'objets ou de phrases importantes, qu'il s'agisse de joueurs, de scores ou de tout autre élément considéré contextuellement comme un point central de l'article en question.
```

**Review by Biokax — CHANGES_REQUESTED** [review]

**Comment by TicClick** [comment]
> I am aware of this, but I have translated the article for information purposes. I will add a mention at the top of the article about this point.

why? my comment (which I don't find worth the reader's attention) was questioning the purpose of the translation instead

**Comment by Biokax** [comment]
> while the initiative is good, I highly doubt we'll ever have news in any language other than English

in the translation its say that news need to be write in english, so I think its good

**Comment by Walavouchey** [comment]
@TicClick Regardless of the fact that news are English-only, I agree that the criteria can be translated just for trivia/informational purposes, because these criteria articles are wiki articles just the same as any other article. Why would the news criteria be in the wiki if they can't be available in other languages?

The community meetings article is also translated even though they are only held in English, although I'm not sure if they have a similar note about this in the translations (probably should).

File: wiki/News_Styling_Criteria/fr.md (hunk 3 above)
Comment (DoryanR) [inline wiki/News_Styling_Criteria/fr.md:53]:
```suggestion
Les noms des fichier sont le titre de l'article lui-même avec tous les espaces remplacés par des traits d'union (`-`), et tous les caractères supplémentaires tels que les points (`.`), les points d'exclamation (`!`), les points d'interrogation (`?`), les virgules (`,`), les apostrophes (`'`), les guillemets (`"`), moins que (`<`), plus que (`>`), les deux-points (`:`), les barres obliques (`/`), les barres obliques (`\`), les barres verticales (`|`) et les astérisques (`*`) sont retirés.
```

File: wiki/News_Styling_Criteria/fr.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+### Noms de fichiers d'articles
+
+Les articles de news sont des fichiers markdown (``.md``) nommés selon le format suivant :
+
+``yyyy-mm-dd-<article title>.md``
+
+Où `yyyy` est l'année à 4 chiffres, `mm` est le mois à 2 chiffres, et `dd` est la date à 2 chiffres.
+
+Les noms des fichier sont le titre de l'article lui-même avec tous les espaces remplacés par des traits d'union (`-`), et tous les caractères supplémentaires tels que les points (`.`), les points d'exclamation (`!`), les points d'interrogation (`?`), les virgules (`,`), les apostrophes (`'`), les guillemets (`"`), moins que (`<`), plus que (`>`), les deux-points (`:`), les barres obliques (`/`), les barres obliques (`\`), les barres verticales (`|`) et les astérisques (`*`) sont retirés.
```
Comment (Biokax) [inline wiki/News_Styling_Criteria/fr.md:53]:
```suggestion
Les noms des fichiers sont le titre de l'article lui-même avec tous les espaces remplacés par des traits d'union (`-`), et tous les caractères supplémentaires tels que les points (`.`), les points d'exclamation (`!`), les points d'interrogation (`?`), les virgules (`,`), les apostrophes (`'`), les guillemets (`"`), moins que (`<`), plus que (`>`), les deux-points (`:`), les barres obliques (`/`), les barres obliques (`\`), les barres verticales (`|`) et les astérisques (`*`) sont retirés.
```

**Review by Satsukel — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**